dso-l2/src/main/java/com/tc/objectserver/handler/ProcessTransactionHandler.java

```diff
@@ -106,6 +108,12 @@ public synchronized void reconnectComplete() {
     public void handleEvent(TCMessage context) throws EventHandlerException {
       NodeID destinationID = context.getDestinationNodeID();
       invokeReturn.remove((ClientID)destinationID, context);
+      if(context instanceof VoltronEntityMultiResponse) {
```

Comment:
AFAIK there are two types of messages getting added into multiSend stage `VoltronEntityMultiResponse` and `VoltronEntityAppliedResponse`, we are handling other type (`VoltronEntityAppliedResponse`) at line no `323`. Correct me if I missed anything.